command/edit.go

```diff
@@ -0,0 +1,136 @@
+package command
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+
+	cli "gopkg.in/urfave/cli.v1"
+	log "github.com/Sirupsen/logrus"
+	yaml "gopkg.in/yaml.v2"
+	vault "github.com/seatgeek/hashi-helper/command/vault"
+)
+
+type preferences map[string]preference
+type preference struct {
+	KeybaseTeam  string `yaml:"keybase-team-name"`
```

Comment:
can you `go fmt` all the files please?

Reply:
Yes, apologies. First experience with go.